components/icons/images/tier1/insights-portal.svg

```diff
@@ -1,4 +1,4 @@
 <svg width="18" height="18" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 18 18">
-	<path fill="#494c4e" d="M7.501 7.885a.516.516 0 01-.155.39.533.533 0 01-.39.164H.548a.512.512 0 01-.47-.267.533.533 0 01-.064-.395c.026-.122.039-.246.062-.369C.804 3.98 2.814 1.602 6.11.278c.246-.105.5-.194.758-.265a.477.477 0 01.527.221c.077.094.112.2.107.32v7.33z" fill="#494C4E"/>
-	<path fill="#494c4e" d="M18 9.349V9.5c.008 2.35-.819 4.357-2.481 6.019S11.851 18.009 9.5 18c-2.835-.03-5.123-1.163-6.864-3.4a9.363 9.363 0 01-1.094-1.83 5.65 5.65 0 01-.507-1.33c-.068-.379-.089-.996.45-.996h7.543a.449.449 0 00.335-.136.449.449 0 00.137-.336v-8.5c-.005-.299.142-.456.441-.472.315.01.628.046.938.106 2.27.376 4.09 1.488 5.463 3.336A8.554 8.554 0 0118 9.349z" fill="#494C4E"/>
+	<path fill="#494c4e" d="M7.501 7.885a.516.516 0 01-.155.39.533.533 0 01-.39.164H.548a.512.512 0 01-.47-.267.533.533 0 01-.064-.395c.026-.122.039-.246.062-.369C.804 3.98 2.814 1.602 6.11.278c.246-.105.5-.194.758-.265a.477.477 0 01.527.221c.077.094.112.2.107.32v7.33z" />
+	<path fill="#494c4e" d="M18 9.349V9.5c.008 2.35-.819 4.357-2.481 6.019S11.851 18.009 9.5 18c-2.835-.03-5.123-1.163-6.864-3.4a9.363 9.363 0 01-1.094-1.83 5.65 5.65 0 01-.507-1.33c-.068-.379-.089-.996.45-.996h7.543a.449.449 0 00.335-.136.449.449 0 00.137-.336v-8.5c-.005-.299.142-.456.441-.472.315.01.628.046.938.106 2.27.376 4.09 1.488 5.463 3.336A8.554 8.554 0 0118 9.349z" />
 </svg>
```

Comment:
What's up with this lil guy?

Reply:
I'm not sure. Doing some searching. Seems like it's maybe a github issue with `</svg>` - it's happening in other icons in previous PRs as well (e.g., [this one](https://github.com/BrightspaceUI/core/pull/4014/files#diff-bcb8a7d64cf3992da7d126cea837657a41c4006e217e102d6230c34f4faef38a))